app/main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
```

Comment:
Are you sure the "skills" should be an instance attribute? 
I mean, maybe, it should be stored like a class variable? For example:
class A:
   some_list = []

Reply:
Not really - stick to `checklist.md` please. Instance attribute is correct here